geppetto.js/geppetto-ui/src/3d-canvas/Canvas.js

```diff
@@ -78,8 +78,8 @@ class Canvas extends Component {
     }
 
     if (prevProps !== this.props) {
-      const { data, cameraOptions, threeDObjects } = this.props;
-      await this.threeDEngine.update(data, cameraOptions, threeDObjects, this.shouldEngineTraverse());
+      const { data, cameraOptions, threeDObjects, backgroundColor } = this.props;
```

Comment:
what about the "new" `captureOptions` , shouldn't they also be added here and later below?

Reply:
@afonsobspinto question for you I think

Reply:
Not needed.
captureOptions is only required by ThreeDEngine at instantiation time. 
(Because we need to set preserveDrawingBuffer to True when we want the canvas/scene to provide the capture functionality, we only do that if the captureOptions are set because preserveDrawingBuffer can affect the scene performance)